### ISSUE TITLE
with typescript

### ISSUE THREAD

**Issue by sarmad006** [Wallet-Chat/widgetdemo#2]
Can't we use it with typescript ?!

**Comment by cryptoKevinL** [comment]
Yes, you can - in `react-wallet-chat-V0` you might have to add the .d.ts file.  in our newer version `react-wallet-chat` this should all be included.
